### PR TITLE
[Stream] Added CSP item to FAQ list

### PR DESCRIPTION
**Pull request by dcpena**
Added item to FAQ list to address Content Security Policy (CSP) directives.

### CODE REVIEW

**Review by kodster28 — APPROVED** [review]

File: content/stream/faq/index.md [hunk 1]
```diff
@@ -17,6 +17,18 @@ meta:
 
 Cloudflare decides on which bitrate, resolution, and codec is best for you. We deliver all videos to industry standard H264 codec. We use a few different adaptive streaming levels from 360p to 1080p to ensure smooth streaming for your audience watching on different devices and bandwidth constraints.
 
+### What domains do I need to add to my allowlist if I use Content Security Policy (CSP) directives on my website?
```
Comment (irvinebroque) [inline content/stream/faq/index.md]: Let's make this the last item in the FAQ for Stream (right above Stream Live)

File: content/stream/faq/index.md [hunk 2]
```diff
@@ -17,6 +17,18 @@ meta:
...
+
+If your website uses Content Security Policy (CSP) directives and you use the provided Stream Player, `videodelivery.net`, `*.cloudflarestream.com` and `*.cloudflarestream.com` must be included in the `frame-src` or `default-src` directive to allow the player's `<iframe>` element to load.
```
Comment (irvinebroque) [inline content/stream/faq/index.md]: Can remove the duplicate `*.cloudflarestream.com` (my mistake in original doc)

File: content/stream/faq/index.md [hunk 3]
```diff
@@ -17,6 +17,18 @@ meta:
...
+
+```bash
```
Comment (irvinebroque) [inline content/stream/faq/index.md]: http

File: content/stream/faq/index.md [hunk 4]
```diff
@@ -17,6 +17,18 @@ meta:
...
+Content-Security-Policy: frame-src; 'self' *.videodelivery.net *.cloudflarestream.com *.cloudflarestream.com;
```
Comment (irvinebroque) [inline content/stream/faq/index.md]: Remove extra `*.cloudflarestream.com`

File: content/stream/faq/index.md [hunk 5]
```diff
@@ -17,6 +17,18 @@ meta:
...
+```
+
+If your website uses CSP directives and you use your own player, `videodelivery.net`, `*.cloudflarestream.com` and `*.cloudflarestream.com` must be included in the `media-src`, `img-src`, and `connect-src` directives, or the fallback directive, `default-src`.
```
Comment (irvinebroque) [inline content/stream/faq/index.md]: Remove dupe

File: content/stream/faq/index.md [hunk 6]
```diff
@@ -17,6 +17,18 @@ meta:
...
+
+If you need to ensure only videos from your Cloudflare Stream account can be played on your website, replace `*` in `*.cloudflarestream.com` in the examples above with your unique customer code, which can be found in the Stream Dashboard. This subdomain is unique to your Cloudflare Stream account.
```
Comment (irvinebroque) [inline content/stream/faq/index.md]: Let's omit this for now.

File: content/stream/faq/index.md (hunk 1 above)
Comment (dcpena) [reply content/stream/faq/index.md]: Done

File: content/stream/faq/index.md (hunk 2 above)
Comment (dcpena) [reply content/stream/faq/index.md]: Removed

File: content/stream/faq/index.md (hunk 4 above)
Comment (dcpena) [reply content/stream/faq/index.md]: Removed

File: content/stream/faq/index.md (hunk 6 above)
Comment (dcpena) [reply content/stream/faq/index.md]: Done

File: content/stream/faq/index.md (hunk 3 above)
Comment (dcpena) [reply content/stream/faq/index.md]: Fixed

File: content/stream/faq/index.md (hunk 5 above)
Comment (dcpena) [reply content/stream/faq/index.md]: Done

**Review by irvinebroque — APPROVED** [review]
🚀

**Pull request merged**